resources/fetchers/iam_fetcher.go

```diff
@@ -60,3 +60,13 @@ func (r IAMResource) GetID() (string, error) {
 func (r IAMResource) GetData() interface{} {
 	return r.Data
 }
+
```

Comment:
You did not remove `GetID()` in `iam_fetcher.go`

Reply:
Thanks 💪